### DISCUSSION TITLE
Adding the azure tags in the metrics dimension

### DISCUSSION THREAD

**Discussion by taruna-verma** [tomkerkhove/promitor#1429] [Q&A] [answered]
Is it possible to add the azure tags in the metrics dimension.
For example:
-azure resource group.
resource group-1 has tags PlanID= abc, KeyID=123
resource group-1 has tags PlanID= abc, KeyID=124

I want to query postgres metrics as-
azure_postgresql_cpu_percent{planid="abc"}

Is this possible?

**Comment by tomkerkhove** [comment, marked as answer]
This is not supported yet but will come later on and is tracked in https://github.com/tomkerkhove/promitor/issues/599.
